zerver/openapi/zulip.yaml

```diff
@@ -3744,3 +3687,40 @@ components:
           type: string
         default: null
       required: false
+    Narrow:
+      name: narrow
+      in: query
+      description: |
+        A JSON-encoded array of length 2 indicating the narrow for which you'd
+        like to receive events for. For instance, to receive events for the
+        stream `Denmark`, you would specify `narrow=['stream', 'Denmark']`.
+        Another example is `narrow=['is', 'private']` for private messages.
+        Default is `[]`.
+      schema:
+        type: array
+        items:
+          anyOf:
+          - type: string
+          - type: integer
+        default: narrow=[]
```

Comment:
This should be `default: []` to pass validation on openapi-generator. Everything else LGTM.

Reply:
Ah, I missed it. Fixed. Thanks @orientor.